test/multiverse/suites/awssqs/awssqs_instrumentation_test.rb

```diff
@@ -0,0 +1,90 @@
+# This file is distributed under New Relic's license terms.
+# See https://github.com/newrelic/newrelic-ruby-agent/blob/main/LICENSE for complete details.
+# frozen_string_literal: true
+
+class AwssqsInstrumentationTest < Minitest::Test
```

Comment:
I think it would be good to either have 100% branch coverage on the new code added, or at least make sure each `rescue` block defined in the new `AwsSqs` module is reached. (Need some unhappy path tests)

Reply:
added a sad test for each method here, checks the log to make sure the message is there https://github.com/newrelic/newrelic-ruby-agent/pull/2679/commits/f251b1498325f0b1ff4e7073227890e03e985b11